src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age && (
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      )}
+
+      {person.isMarried && (
+        person.sex === 'm'
+          ? <p className="Person__partner">{`${person.partnerName} is my wife`}</p>
+          : <p className="Person__partner">{`${person.partnerName} is my husband`}</p>
```

Comment:
Create a separate variable and use it here to simplify your code

`const partner = person.sex === 'm' ? 'wife' : 'husband'`